sdk/xmake.lua

```diff
@@ -847,12 +843,6 @@ rule("cheriot.firmware")
 			-- loader will fail.  If it is too big, we waste space.
 			thread.trusted_stack_size = loader_trusted_stack_size + (24 * thread.trusted_stack_frames)
 
-			if thread.stack_size > stack_size_limit then
-				raise("thread " .. i .. " requested a " .. thread.stack_size ..
-				" stack.  Stacks over " .. stack_size_limit ..
-				" are not yet supported in the compartment switcher.")
-			end
```

Comment:
Why did you remove the check and not update it with the new, 64 KiB size limit?

Reply:
I removed the check then discovered the new limit.  I'll add it back.